### ISSUE TITLE
communications for maintainers

### ISSUE THREAD

**Issue by bketelsen** [go-commons/commons#6]
We probably need a back-channel communication between organization maintainers.  I created #go-commons on the gopherslack.  Email might be good?  A private google group? Or everything in the open in GH issues (very open and community friendly!)

**Comment by ChrisHines** [comment]
I don't know how involved I will end up being with this project, but I prefer keeping design discussions in GH issues and PRs. Slack is OK for some things, but if it turns into a design discussion it should be required to write it up more formally in GH and allow time for the community to weigh in.

**Comment by markbates** [comment]
I agree with @ChrisHines, but I do think there needs to be a private method of communication, there might be private issues that need to be addressed. I've fine with Slack or Email.

**Comment by stp-ip** [comment]
Kubernetes is currently adopting the rust enhancement proposal style (https://github.com/kubernetes/community/blob/master/contributors/design-proposals/architecture/1-kubernetes-enhancement-proposal-process.md). It might be a bit too heavy for starting off, but it feels like a great read to make decisions on design proposals. With that keeping these in git and versioned seems transparent, contribution friendly and a workable solution.
Slack is a good solution, but it always leaves a salty feel due to using closed solutions to built open communities. Gitter.im might be a compromise as it has a hosted version, a public archive (example: https://gitter.im/gitterHQ/gitter/archives/all) and is open source (https://gitlab.com/gitlab-org/gitter), if Gitlab ever decides to shut it down. Mattermost unfortunately has no known hosted version.

My vote therefore would be discussion in issues + PRs, design proposals in git repo, private communication gitter.im.
